sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/DynamicDestinationsHelpers.java

```diff
@@ -165,8 +173,12 @@ public TableDestination getTable(DestinationT destination) {
     }
 
     @Override
-    Coder<DestinationT> getDestinationCoderWithDefault(CoderRegistry registry)
+    public Coder<DestinationT> getDestinationCoderWithDefault(CoderRegistry registry)
```

Comment:
Was change of modifier here (to public) intentional ? Wasn't sure from your explanation below.

Reply:
Unintentional. Will revert.